docs/grammar/Solidity.g4

```diff
@@ -329,7 +329,6 @@ expression:
 		identifier
 		| literal
 		| elementaryTypeName[false]
-		| userDefinedTypeName
```

Comment:
Why was this removed?

Reply:
Because it is not parsed like that, the expression parser uses `MemberAccess` to parse `.`.